src/js/common/util.js

```diff
@@ -0,0 +1,161 @@
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.devpmt");
+
+/**
+ * binderCheckboxToBoolean
+ *
+ * By default GPII binder, when bound to a checkbox input will
+ * represent checked as an array of one item, and unchecked as
+ * an empty array. For many situations we would like these to
+ * be represented as true and false. These transforms accomplish
+ * that mapping.
+ *
+ * @param (Array) value - The array coming from binder, most likely []
```

Comment:
See here for our actual JavaDoc-like format https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/Fluid.js#L339 ( for which you have already written a parser)

Reply:
these have been fixed up